scripts/gui/microbe_hud.js

```diff
@@ -93,7 +105,10 @@ function runMicrobeHUDSetup(){
             compoundHydrogenSulfide: randomBetween(0, hydrogenSulfide),
             HydrogenSulfideMax: hydrogenSulfide,
         });
-
+		
+		// Pseudo population code
+		document.getElementById("populationCount").textContent = 27;
```

Comment:
For consistency this should call `updatePopulation(randomBetween(0, 50))` with a randomly generated number.